File: src/jp/senchan/android/wasatter/util/OAuthPreference.java
/**
 *
 */
package jp.senchan.android.wasatter.util;

import jp.senchan.android.wasatter.R;
import android.content.Context;
import android.preference.Preference;
import android.util.AttributeSet;

/**
 * @author Senka/Takuji
 *
 */
public class OAuthPreference extends Preference {

	public OAuthPreference(Context context, AttributeSet attrs, int defStyle) {
		super(context, attrs, defStyle);
		// TODO 自動生成されたコンストラクター・スタブ
	}

	public OAuthPreference(Context context, AttributeSet attrs) {
		super(context, attrs);
		// TODO 自動生成されたコンストラクター・スタブ
	}

	public OAuthPreference(Context context) {
		// TODO 自動生成されたコンストラクター・スタブ
		super(context);
		this.setTitle(R.string.title_setting_twitter_oauth_token_get);
		this.setSummary(R.string.summary_setting_twitter_oauth_token_get);
	}
}

File: app/src/main/java/jp/senchan/android/wasatter/activity/OAuthPreference.kt
/**
 *
 */
package jp.senchan.android.wasatter.activity

import android.content.Context
import android.preference.Preference
import android.util.AttributeSet
import jp.senchan.android.wasatter.R

/**
 * @author Senka/Takuji
 */
class OAuthPreference : Preference {
    constructor(context: Context?, attrs: AttributeSet?, defStyle: Int) : super(context, attrs, defStyle) { // TODO 自動生成されたコンストラクター・スタブ
    }

    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs) { // TODO 自動生成されたコンストラクター・スタブ
    }

    constructor(context: Context?) : super(context) { // TODO 自動生成されたコンストラクター・スタブ
        this.setTitle(R.string.title_setting_twitter_oauth_token_get)
        this.setSummary(R.string.summary_setting_twitter_oauth_token_get)
    }
}